docs/releases/v2018.1.rst

```diff
@@ -0,0 +1,387 @@
+Gluon 2018.1 (in development)
+#############################
+
+Important notes
+***************
+
+This version changes the flash partition layout on some devices (CPE/WBS 210/510). To avoid
+upgrade failures, make sure to upgrade to Gluon 2017.1.8 or the latest Gluon 2016.2.x (unreleased)
+before installing Gluon 2018.1.
+
+Some of the following paragraphs describe so-called "feature flags". This new concept is
+explained in the :ref:`releases-v2018.1-site-changes` section.
+
+Added hardware support
+**********************
+
+ar71xx-generic
+==============
+
+* ALFA NETWORK
+
+  - AP121F
+
+* AVM
+
+  - Fritz!Box 4020
+
+* OpenMesh
+
+  - A40
+  - A60
+  - OM2P v4
+  - OM2P-HS v2
```

Comment:
20:29 <ecsv_> neoraider: regaring the changelog: the added OM2P-HS in 2018.1.x is the *v4* and not *v2*